tests/roles/manila_adoption/templates/manila_cephfs.yaml.j2

```diff
@@ -0,0 +1,58 @@
+spec:
+  manila:
+    enabled: true
+    apiOverride:
+      route: {}
+    template:
+      databaseInstance: openstack
+      databaseAccount: manila
+      manilaAPI:
+        customServiceConfig: |
+          [DEFAULT]
+          {% if manila_backend == "cephnfs" -%}
+          enabled_share_protocols=nfs
+          {% else -%}
+          enabled_share_protocols=cephfs
+          {%- endif %}
+
+        replicas: 1
+        networkAttachments:
+          - internalapi
+        override:
+          service:
+            internal:
+              metadata:
+                annotations:
+                  metallb.universe.tf/address-pool: internalapi
+                  metallb.universe.tf/allow-shared-ip: internalapi
+                  metallb.universe.tf/loadBalancerIPs: 172.17.0.80
+              spec:
+                type: LoadBalancer
+      manilaScheduler:
+        replicas: 1
+      manilaShares:
+        share1:
+          customServiceConfig: |
+            [DEFAULT]
+            enabled_share_backends=cephfs
+            {% if manila_backend == "cephnfs" -%}
+            host = hostgroup
+            {%- endif %}
```

Comment:
the host must be set to "hostgroup" on both drivers..

Reply:
ack, thanks, good catch, we'll fix it also in [1][2]

[1] https://github.com/openstack-k8s-operators/manila-operator/blob/main/config/samples/backends/ceph-nfs/backend.yaml
[2] https://github.com/openstack-k8s-operators/manila-operator/blob/main/config/samples/backends/cephfs/backend.yaml

Reply:
Fixed for now here.